setup.py

```diff
@@ -171,6 +171,17 @@
     print("32-bit Windows Python runtime is not supported. Please switch to 64-bit Python.")
     sys.exit(-1)
 
+import platform
+python_min_version = (3, 6, 1)
+python_min_version_str = '.'.join((str(num) for num in python_min_version))
+python_max_version = (3, 9, 0)
+python_max_version_str = '.'.join((str(num) for num in python_max_version))
```

Comment:
I'm surprised there isn't a more compact way to write this code

Reply:
me, too (not I'm aware of)

Reply:
https://pypi.org/project/semver/ may make the code simple, but need to introduce a new dependency...

Reply:
How about the following?
```suggestion
python_max_version_str = '.'.join(map(str, python_max_version))
```